features/time-relative-selectors.yml

```diff
@@ -0,0 +1,8 @@
+name: Time-dimensional pseudo-selectors
```

Comment:
Neither the name nor the description has terminology in common with the file name. Can something change here to make them line up with each other? I kinda like "time-relative" as being more approachable, but either is fine.

Reply:
Good point- I agree that time-relative is more approachable and went with that.